test/e2e/set/scyllaoperatorconfig/config.go

```diff
@@ -0,0 +1,35 @@
+package scyllaoperatorconfig
+
+import (
+	"time"
+
+	scyllav1alpha1 "github.com/scylladb/scylla-operator/pkg/api/scylla/v1alpha1"
+	"github.com/scylladb/scylla-operator/pkg/gather/collect"
+	"github.com/scylladb/scylla-operator/pkg/naming"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+const (
+	testTimeout = 15 * time.Minute
+)
+
+var (
+	socResourceInfo = collect.ResourceInfo{
+		Resource: schema.GroupVersionResource{
+			Group:    "scylla.scylladb.com",
+			Version:  "v1alpha1",
+			Resource: "scyllaoperatorconfigs",
+		},
+		Scope: meta.RESTScopeRoot,
+	}
+)
+
+func GetDefaultSOC() *scyllav1alpha1.ScyllaOperatorConfig {
+	return &scyllav1alpha1.ScyllaOperatorConfig{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: naming.SingletonName,
+		},
+	}
+}
```

Comment:
Test resources are kept in fixture package

Reply:
it's an empty definition but done